fern/openapi-ir/definition/ir.yml

```diff
@@ -0,0 +1,136 @@
+types: 
+  IntermediateRepresentation: 
+    properties: 
+      displayName: optional<string>
+      environments: list<Environment>
```

Comment:
this is the `servers` block right? i'd just use their names, makes it easy to externalize if we want to

Reply:
yup it is